### PR TITLE
AUTH-482: set required-scc for openshift workloads

### PR DESCRIPTION
**Pull request by liouk**

### CODE REVIEW

File: bindata/network/node-identity/managed/node-identity.yaml [hunk 1]
```diff
@@ -29,6 +29,7 @@ spec:
       annotations:
         hypershift.openshift.io/release-image: {{.ReleaseImage}}
         target.workload.openshift.io/management: '{"effect": "PreferredDuringScheduling"}'
+        openshift.io/required-scc: hostnetwork-v2
```
Comment (kyrtapz) [inline bindata/network/node-identity/managed/node-identity.yaml:32]: node-identity in managed is not a hostNetworked pod, this should work with restricted-v2?

Comment (liouk) [reply bindata/network/node-identity/managed/node-identity.yaml]: Thanks for this -- our monitor tests probably run only on self-hosted therefore I have missed this case. Fixed :+1:

**Pull request merged**